sdk/dynamicconfig/metric/controller/notify/metricconfig.go

```diff
@@ -0,0 +1,36 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package notify
+
+import (
+	"errors"
+
+	pb "github.com/open-telemetry/opentelemetry-proto/gen/go/experimental/metricconfigservice"
+)
+
+// A configuration used in the SDK to dynamically change metric collection and tracing.
+type MetricConfig struct {
```

Comment:
How much benefit does this MetricConfig class bring? It only has a Validate() function which can be a simple free function.

Reply:
Great question, this class was another holdover from the original code. The changes removed some of its other functionality, though I figured it might still be a nice abstraction layer over the protocol details. But if it only has `Validate()`, I agree it's probably a good idea to go ahead and remove it.